mreg_cli/api/models.py

```diff
@@ -384,6 +390,11 @@ class NameServer(FrozenModelWithTimestamps, WithTTL):
     id: int  # noqa: A003
     name: str
 
+    @classmethod
+    def endpoint(cls) -> Endpoint:
+        """Return the endpoint for the class."""
+        return Endpoint.Nameservers
```

Comment:
Had to be added since `WithTTL` now inherits from `APIMixin`, which requires that an endpoint be defined on its subclasses.